src/kagglehub/config.py

```diff
@@ -58,6 +60,12 @@ def get_kaggle_api_endpoint() -> str:
 
 
 def get_kaggle_credentials() -> Optional[KaggleApiCredentials]:
+    global _kaggle_credentials
+
+    # Check for credentials in the global variable
+    if _kaggle_credentials.username != None and _kaggle_credentials.key != None:
```

Comment:
Won't this throw a `KeyError` if `_kaggle_credentials` is None?

Replace with: `if _kaggle_credentials:`

Reply:
i cant just have _kaggle_credentials because it doesnt check the fields inside _kaggle_credentials. i think this should fix both problems